outputs.tf

```diff
@@ -12,3 +12,9 @@ output "bucket" {
   description = "S3 bucket subscribed to Observe Lambda"
   value       = local.s3_bucket
 }
+
+# experimental: do not use yet
```

Comment:
Include this in the description, if anything. How would an output be experimental though, and what is the criteria for removal of that status?

Reply:
i added it in the description. basically it's only used in a test. i'm not sure what our plans are for exposing almost all resources as outputs